fullstack-cert/html-projects/framed-videos/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css"/> 
+  <title>Display Videos in an iframe</title>
+</head>
+<body>
+  <h1>How to display videos in an iframe</h1>
+  <br>
+  <iframe 
+  width="560"
```

Comment:
this might have been pulled from YouTube but the `frameborder` and `align` attributes are deprecated.

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/iframe

so those should be removed from out final solution here.